ostelco-ios-client/ViewControllers/SignUp/TheLegalStuffViewController.swift

```diff
@@ -15,25 +15,25 @@ class TheLegalStuffViewController: UIViewController {
         case termsAndConditions = "https://pi-redirector.firebaseapp.com/terms-and-conditions"
     }
     
-    @IBOutlet weak var termsAndConditionsLabel: UILabel!
-    @IBOutlet weak var privacyPolicyLabel: UILabel!
-    @IBOutlet weak var oyaUpdatesLabel: UILabel!
+    @IBOutlet private weak var termsAndConditionsLabel: UILabel!
```

Comment:
Is this a usual practice to add `private` to @IBoutlet ?

Reply:
yes - otherwise outside classes can re-set the label, which you don't want. 

For things like cells it's more common to use `private(set)`, so you can set the `text` on a label, but not the the label itself.